.clang-tidy

```diff
@@ -0,0 +1,2 @@
+Checks: '-*,kokkos-*'
+FormatStyle: file
```

Comment:
Isn't this redundant? I mean if it's not looking for the file anyway, it doesn't know that it should use the file.

Reply:
It is not.  This is meant if you instruct clang-tidy to fix the error founds via the flag `-fix` (when the check supports it)